scripts/network-init

```diff
@@ -108,21 +108,14 @@ rename_network_label() {
     done
 }
 
-maybe-enable-tls-verification() {
-  verification_enabled="$(${XE} pool-list params=tls-verification-enabled --minimal)"
-  if [ "$FIRSTBOOT_ENABLE_TLS_VERIFICATION" == "true" ] && [ "$verification_enabled" != "true" ] ; then
-      ${XE} pool-enable-tls-verification
-  fi
-}
-
 if [ "$UPGRADE" = "true" ]; then
     rename_network_label
 else
     prepare_networking
     rename_network_label
     # Ensure changes are synced to disk
     xe pool-sync-database
-    maybe-enable-tls-verification
+    xe pool-enable-tls-verification
```

Comment:
What if it is an 'update' (i.e. installed as a hotfix in XenCenter) as opposed to an upgrade (reboot into host-installer). Which path will this script take here?